keyvault/data-plane/pom.xml

```diff
@@ -293,13 +293,13 @@
             <configuration>
                 <properties>
                     <property>
-<name>KEYVAULT_VAULTURI</name>
-       		<value>https://tifchen-keyvault-fancy.vault.azure.net</value>
+                          <name>KEYVAULT_VAULTURI</name>
+                          <value>https://azure-keyvault-3.vault.azure.net</value>
```

Comment:
Looks like you are setting these in 3 places. One is here, one is below and one in the build definition. Do they need to be set in all these places?

Reply:
Updated, the values should only be in environment variables section of pom.
Removed from other places.